source/SpinalHDL/Simulation/install/VCS.rst

```diff
@@ -21,6 +23,9 @@ Prepend the following paths to your ``LD_LIBRARY_PATH``
   export LD_LIBRARY_PATH=$VERDI_HOME/share/PLI/Ius/LINUX64:$LD_LIBRARY_PATH 
   export LD_LIBRARY_PATH=$VERDI_HOME/share/PLI/MODELSIM/LINUX64:$LD_LIBRARY_PATH 
 
+For those who encounter the problem of ``Compilation of SharedMemIface.cpp failed``, make sure that you have installed C++ boost library correctly.
+The header files and the library files path are added to ``CPLUS_INCLUDE_PATH``, ``LIBRARY_PATH`` and ``LD_LIBRARY_PATH`` respectively.
```

Comment:
Maybe i understood the frase incorrectly. Is my suggestion correct? If so evaluate what version do you prefer :D
```suggestion
The header and library files path should be added to ``CPLUS_INCLUDE_PATH``, ``LIBRARY_PATH`` and ``LD_LIBRARY_PATH`` respectively.
```

Reply:
Great! Thanks for your suggestion.